app/views/file_attachments/edit.html.erb

```diff
@@ -1,18 +1,27 @@
 <% content_for :title, t("file_attachments.edit.title", title: @edition.title_or_fallback) %>
-<% content_for :back_link, render_back_link(href: file_attachments_path(@edition.document),
+
+<% back_link_path = if params[:wizard] == "featured"
+                      featured_attachments_path(@edition.document)
+                    else
+                      file_attachments_path(@edition.document)
+                    end %>
+
+<% content_for :back_link, render_back_link(href: back_link_path,
                                             data_attributes: { "modal-action": "back" }) %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
+  <div class="<%= rendering_context == 'modal' ? 'govuk-grid-column-full' : 'govuk-grid-column-two-thirds' %>">
+    <% data = {}
+       data["modal-action"] == "update" if rendering_context == "modal" %>
```

Comment:
(I think we can do the same here, as it's only used once, in a modal context.)

Reply:
@benthorner Do the same here as?

Reply:
Ah, I was referring to [this comment, a few lines above](https://github.com/alphagov/content-publisher/pull/1901#discussion_r392351267). I'll remember to be more precise next time 👌.

Reply:
Yeah. I think those changes are in a different commit, so they didn't appear together.

Reply:
I'll remove this commit and rebase.